### DISCUSSION TITLE
Add Turbopack to existing project

### DISCUSSION THREAD

**Discussion by FleetAdmiralJakob** [vercel/turborepo#4498] [Help] [answered]
How can I migrate my existing project to Turbopack?

**Comment by MonstraG** [comment, marked as answer]
Had to download quickstart to find out that arcane knowledge:

just add `--turbo` to your `dev` script: `"dev": "next dev --turbo",`.

**Comment by FleetAdmiralJakob** [comment]
What is with the build and start script?

**Reply by MonstraG** [reply]
They basically say that you cannot use it for building yet, only for dev.

**Reply by FleetAdmiralJakob** [reply]
Cool. Thx
